src/cli/main.py

```
@@ -0,0 +1,211 @@
+
+@cli.command()
+@click.option("--title", required=True, help="Book title")
+@click.option(
+    "--preset",
+    type=click.Choice(["small", "medium", "large"]),
+    default="medium",
+    help="Capture region preset"
+)
+@click.option("--interactive", is_flag=True, help="Use interactive UI for region selection")
+@click.pass_obj
+def calibrate(config, title: str, preset: str, interactive: bool) -> None:
```

Ruff (0.12.2)

Redefinition of unused `calibrate` from line 11

(F811)